var/spack/repos/builtin/packages/py-flask-babel/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFlaskBabel(PythonPackage):
+    """Implements i18n and l10n support for Flask."""
+
+    homepage = "https://pythonhosted.org/Flask-Babel/"
+    pypi     = "Flask-Babel/Flask-Babel-2.0.0.tar.gz"
+
+    version('2.0.0',  sha256='f9faf45cdb2e1a32ea2ec14403587d4295108f35017a7821a2b1acb8cfd9257d')
+    version('1.0.0',  sha256='d6a70468f9a8919d59fba2a291a003da3a05ff884275dddbd965f3b98b09ab3e')
+
+    depends_on('python@2.7:2.8,3.6:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@3.5:', when='@2.0.0:', type=('build', 'run'))
    depends_on('python@2.7:2.8,3.6:', when='@1.0.0', type=('build', 'run'))
```

Reply:
Didn't think about this either :-/ To keep things simple, I've removed the older version of the package.